pinot-tools/src/main/java/org/apache/pinot/tools/streams/AirlineDataStream.java

```diff
@@ -125,7 +125,7 @@ public void run() {
             }
 
             TimeFieldSpec spec = pinotSchema.getTimeFieldSpec();
-            String timeColumn = spec.getIncomingTimeColumnName();
+            String timeColumn = spec.getName();
```

Comment:
Conceptually this should be the incoming name

Reply:
changed